src/containers/editor/graph/Toolbar.tsx

```diff
@@ -82,20 +82,24 @@ const Toolbar = memo(({ id }: ToolbarProps) => {
           <ArrowLeft className="icon" />
         </ToolbarButton>
       )}
-      {!isRoot && (
-        <ToolbarButton title={t(foldSiblings ? "fold sibings" : "unfold sibings")} onClick={triggerFoldSiblings}>
-          {foldSiblings ? <CopyMinus className="icon" /> : <CopyPlus className="icon" />}
-        </ToolbarButton>
+      {hasSiblings && (
```

Comment:
I think you should add it for "fold node" rather than "fold siblings". By the way, "fold siblings" is not what the user expects, it should mean to hide sibings and their descendants, instead of only hiding the descendants of siblings (I'll fix it later). 😄

Reply:
This is updated. Thank you.